dnf-behave-tests/features/environment.py

```diff
@@ -106,6 +106,7 @@ def __init__(self, userdata, force_installroot=False):
         self.disable_plugins = True
         self.disable_repos_option = "--disablerepo='*'"
         self.assumeyes_option = "-y"
+        self.cwd = None
```

Comment:
Not sure `cwd` is actually a convention to call the working directory, I'd prefer to call it e.g. `work_dir`, I myself wouldn't know what `cwd` is.